sdk/python/kfp/_client.py

```diff
@@ -186,8 +190,12 @@ def _extract_pipeline_yaml(self, package_file):
         if len(all_yaml_files) == 0:
           raise ValueError('Invalid package. Missing pipeline yaml file in the package.')
 
-        if len(all_yaml_files) > 1:
-          raise ValueError('Invalid package. Multiple yaml files in the package.')
+        pipeline_file = all_yaml_files[0]
+        if 'pipeline.yaml' in all_yaml_files:
+          pipeline_file = 'pipeline.yaml'
+        else:
+          if len(all_yaml_files) > 1:
+            raise ValueError('Invalid package. There is no pipeline.yaml file and there are multiple yaml files.')
 
```

Comment:
seems both the code blocks are doing exactly same thing - can it be function instead _get_pipeline_file_ ?

Reply:
+1

Reply:
Done.